afutests/afp/ocxl_afp3.c

```diff
@@ -564,6 +517,17 @@ int main(int argc, char *argv[])
 		       bpc_tb_cnt6, bw_tb_cnt6);
 		printf("No cred cycles  %016lx %016lx %016lx %#12.8f %#1.8f %#12.8f\n", count7, count7_prev, delta_cnt7,  bw_cnt7,
 		       bpc_tb_cnt7, bw_tb_cnt7);
+		printf("\n");
+
+		count0_prev = count0;
+		count1_prev = count1;
+		count2_prev = count2;
+		count3_prev = count3;
+		count4_prev = count4;
+		count5_prev = count5;
+		count6_prev = count6;
+		count7_prev = count7;
+		c0Time_prev = c0Time;
```

Comment:
I wonder if these would be better off refactored into an array of previous counts & looped over